internal/learn/future/shimmer.md

```diff
@@ -1,37 +1,30 @@
 
-## Pre-launch staking
-
-During 90 days before the launch of the Shimmer network, you will be able to stake your IOTA tokens to receive SMR tokens on the Shimmer network upon launch. The IOTA Foundation will announce the start date 24 hours in advance. During these 90 days, you will earn 1 SMR for every 1Mi IOTA you staked every 10 seconds. There is no risk involved in staking your IOTA tokens. You will remain in control of your tokens and won't have to transfer them to a third party. Let's take a closer (simplified) look at the technology behind the process.
-
-### Technical overview
-
-Staking will happen using the official IOTA [Firefly Wallet](https://firefly.iota.org/). To stake your IOTA tokens in return for SMR tokens, you create a special staking transaction with a specific payload. This declares that you want to stake that amount of IOTA tokens to receive SMR tokens. The Hornet node uses a plugin to register which addresses have staked IOTA and for how long - node owners can opt in to enable this plugin.
-
-Those who are already familiar with the upcoming Build/Burn Vote process will recognize the similarities between that voting process and the Shimmer staking process, as the same Hornet plugin performs it.
-
-Here is **an example** of how this would work: Bob has 1Mi on an address and uses Firefly, the official IOTA wallet. He wants to use his IOTA tokens to stake for some Shimmer tokens (SMR). He clicks the stake button in Firefly, which generates the staking transaction. Hornet sees this message and starts monitoring how long the IOTA tokens are staked on that address. If the tokens are moved from that address, then the staking period for that address ends.
+## Post-launch staking
 
-For every milestone, 1 Mi staked returns 1 SMR token. So after the first milestone, the returned amount will be 1 SMR. Ten seconds later, after the second milestone, it will be 2 SMR and so on. As long as his IOTA tokens stay on the address used for staking, the number of SMR tokens will accumulate.
+After the Shimmer network launches, you will be able to stake your SMR tokens directly to receive more SMR tokens. An initial annual percentage yield(APY) of 8% is planned. This can be changed later by the community. More information will follow on "post-launch" staking.
```

Comment:
```suggestion
After the Shimmer network launches, you will be able to stake your SMR tokens directly to receive more SMR tokens. An initial annual percentage yield (APY) of 8% is planned. This can be changed later by the community. More information will follow on post-launch staking.
```

I guess we can do without the quotes, as it is a grammatically correct description of a time period.